### PR TITLE
feat(component): Edit Component page

### PR DESCRIPTION
**Pull request by tuannn2**
Changes:
   Feature edit Component
         - Tab Sumary
         - Tab Release
         - Tab Attachments

### CODE REVIEW

File: src/app/[locale]/components/edit/[id]/detail.module.css
Comment (hoangnt2) [inline src/app/[locale]/components/edit/[id]/detail.module.css:1]: remove unused css

File: src/app/[locale]/components/edit/[id]/EditComponent.tsx
Comment (hoangnt2) [inline src/app/[locale]/components/edit/[id]/EditComponent.tsx:1]: create components folder to store this file

File: src/app/[locale]/components/edit/[id]/detail.module.css
Comment (hoangnt2) [reply src/app/[locale]/components/edit/[id]/detail.module.css:1]: Change file name to edit.module.css

File: src/components/Attachments/EditAttachments.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+                componentData={componentData}
+                setComponentData={setComponentData}
+                attachmentUpload={attachmentData}
+                setAttachmentFromUpload={setAttachmentData}
+                show={dialogOpenSelectAttachment}
+                setShow={setDialogOpenSelectAttachment}
+                session={session}
+                onReRender={handleReRender}
+            />
+            {attachmentData ? (
```
Comment (hoangnt2) [inline src/components/Attachments/EditAttachments.tsx:101]: always true

File: src/components/Attachments/EditAttachments.tsx [hunk 2]
```diff
@@ -0,0 +1,136 @@
...
+                <div className={`row ${styles['attachment-table']}`}>
+                    <TiltleAttachment />
+                    <TableAttachment
+                        data={attachmentData}
+                        setAttachmentData={setAttachmentData}
+                        setAttachmentToComponentData={setAttachmentToComponentData}
+                    />
+                    &nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;&nbsp;
```
Comment (hoangnt2) [inline src/components/Attachments/EditAttachments.tsx:109]: use css

File: src/components/Attachments/SelectAttachment/SelectAttachment.tsx [hunk 3]
```diff
@@ -0,0 +1,167 @@
...
+    componentData,
+    setComponentData,
+}: Props) => {
+    const t = useTranslations(COMMON_NAMESPACE)
+    const [files, setFiles] = useState([])
+    const inputRef = useRef(null)
+
+    const handleFileChange = (e: React.ChangeEvent<HTMLInputElement>) => {
+        const files = Array.prototype.slice.call(e.target.files)
+        console.log(files)
```
Comment (hoangnt2) [inline src/components/Attachments/SelectAttachment/SelectAttachment.tsx:50]: remove console.log

File: src/components/Attachments/SelectAttachment/SelectAttachment.tsx [hunk 4]
```diff
@@ -0,0 +1,167 @@
...
+        if (files.length > 0) {
```
Comment (hoangnt2) [inline src/components/Attachments/SelectAttachment/SelectAttachment.tsx:51]: `
 if (files.length > 0) {
            setFiles(files)
        } else {
            setFiles([])
        }
`
to
`setFiles(files)`

File: src/components/Attachments/TableAttachment/TableAttachment.tsx [hunk 5]
```diff
@@ -0,0 +1,149 @@
...
+                                &nbsp;&nbsp;&nbsp;
+                                <input
+                                    className={`${styles['input-comment']}`}
+                                    type='text'
+                                    value={item.checkedComment}
+                                    name='checkedComment'
+                                    onChange={(e) => handleInputChange(e, j)}
+                                    placeholder='Enter Comment'
+                                />
+                                &nbsp;&nbsp;
```
Comment (hoangnt2) [inline src/components/Attachments/TableAttachment/TableAttachment.tsx:126]: use css

**Comment by tuannn2** [comment]
Thank you for your reviewing, @hoangnt2
I have updated.

**Review by heliocastro — COMMENTED** [review]
Can you. please split this commit in small atomic commits ?
Same as said to other previous pull requets. Is too big to review.

Please, try to do in small moves.
Example. messages can be done in a independent commit,

**Pull request closed without merging**